samples/buildpacks/stacks/bionic/base/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:bionic
+ARG cnb_uid=1000
+ARG cnb_gid=1000
+
+RUN apt-get update && \
+  apt-get install -y xz-utils ca-certificates && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN groupadd cnb --gid ${cnb_gid} && \
+  useradd --uid ${cnb_uid} --gid ${cnb_gid} -m -s /bin/bash cnb
+
+ENV CNB_USER_ID=${cnb_uid}
+ENV CNB_GROUP_ID=${cnb_gid}
+
+COPY ./bin/jq-linux64 /usr/local/bin/jq
```

Comment:
I'd prefer not to include binaries in our repo if we can help it. Is there any package we can use for these instead or a CURL command maybe?

Also, does the base image need these or are they more for the build step?

Reply:
Totally agreed. I'll remove.